### PR TITLE
Copy/Adapt nitro_config + automatically run specs

### PR DESCRIPTION
**Pull request by that-jill**
This PR sets up the following:

* brings in nitro_config as a package (and then subsequently modifies it to do the following):
* adds documentation to both `docs/README.md` and in `portal.yml`
* sets up CI with rspec, rubocop, and license_finder

### CODE REVIEW

**Comment by that-jill** [comment]
Converting to draft because I may as well do more work on this branch to set up automated linting, licensing, and publishing.

File: .github/workflows/nitro-config.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  #     - uses: actions/checkout@v3
+  #     - id: version-changed
+  #       uses: tj-actions/changed-files@v22
+  #       with:
+  #         files: ${{ env.PROJECT_DIR }}/**/version.rb
+  #     - uses: xjunior/publish-rubygems-action@master
+  #       if: steps.version-changed.outputs.any_changed == 'true'
+  #       env:
+  #         GITHUB_TOKEN: ${{ secrets.github_token }}
+  #         RUBYGEMS_API_KEY: ${{ secrets.rubygems_api_key }}
```
[the remark this answers is not in the document]
Comment (benlangfeld) [reply .github/workflows/nitro-config.yml:45]: `GITHUB_TOKEN` is automagic. I set `RUBYGEMS_API_KEY`.

[the remark this answers is not in the document]
Comment (that-jill) [reply .github/workflows/nitro-config.yml:45]: Thank you!

File: packages/nitro_config/.rubocop_todo.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# on 2022-07-15 13:58:41 UTC using RuboCop version 1.20.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# Configuration parameters: CountComments, CountAsOne, ExcludedMethods, IgnoredMethods.
+# IgnoredMethods: refine
+Metrics/BlockLength:
```
Comment (benlangfeld) [inline packages/nitro_config/.rubocop_todo.yml:12]: https://github.com/powerhome/nitro-id/blob/main/authentication_service/.rubocop.yml#L7-L10

Comment (benlangfeld) [reply packages/nitro_config/.rubocop_todo.yml:12]: https://github.com/powerhome/power_linting/issues/8

**Comment by that-jill** [comment]
This has been thoroughly tested on our monolithic application. As opposed to yielding any errors on the gem side, the only things we've uncovered are on the monolith side.

**Review by xjunior — APPROVED** [review]

**Pull request merged**